src/test/storage.zig

```diff
@@ -90,6 +90,8 @@ pub const Storage = struct {
 
     memory: []align(config.sector_size) u8,
     size: u64,
+    /// Set bits correspond to sectors that have been written to.
+    written: std.DynamicBitSetUnmanaged,
```

Comment:
:pray: Can you remove this? I already this implemented in the `dj-vopr-workload` branch (which is awaiting review) and I don't want to rebase it (and all of the branches that are based on it...).

Reply:
No worries :)

Reply:
I removed those changes, and left some `//TODO(djg)` where they hook into the fuzzers.